bors.toml

```diff
@@ -1,17 +1,19 @@
 status = [
```

Comment:
We don't use bors anymore; this whole file can be removed.

Reply:
Thanks, I removed the file